libs/interfaces/src/lib/design/design-dimension.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum DesignDimensionEnum {
```

Comment:
We just need to make sure that the front end displays 20x20cm so customers dont get confused

Reply:
Yup. Right now I'm implementing raw functionality but I will adjust the design afterwards